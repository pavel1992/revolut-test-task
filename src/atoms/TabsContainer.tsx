import styled from 'styled-components';

import { TABLET_BREAKPOINT } from '../styleConstants/mediaConstants';

export const TabsContainer = styled.div`
  display: flex;
  flex-direction: row;

  @media(max-width: ${TABLET_BREAKPOINT}) {
    flex-direction: column;
  }
`;
