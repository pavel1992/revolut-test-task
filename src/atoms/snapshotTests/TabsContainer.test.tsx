import 'jest-styled-components';
import * as React from 'react';
import renderer from 'react-test-renderer';

import { TabsContainer } from '../TabsContainer';

test('renders TabsContainer correctly', () => {
  const render = renderer
    .create(<TabsContainer>111</TabsContainer>)
    .toJSON();
  expect(render).toMatchSnapshot();
});
