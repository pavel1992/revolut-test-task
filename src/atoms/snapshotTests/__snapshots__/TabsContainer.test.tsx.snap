// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TabsContainer correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

@media (max-width:768px) {
  .c0 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

<div
  className="c0"
>
  111
</div>
`;
